pkgs/applications/networking/cluster/terraform-providers/default.nix

```
@@ -2,11 +2,16 @@
 , buildGoPackage
 , fetchFromGitHub
 , callPackage
+, runtimeShell
 }:
 let
-  list = import ./data.nix;
+  list = lib.importJSON ./providers.json;
 
-  toDrv = data:
+  toDrv = name: data:
+    let
+      fallbackProviderSourceAddress = "nixpkgs/${data.owner}/${name}";
+      providerSourceAddress = lib.attrByPath ["provider-source-address"] fallbackProviderSourceAddress data;
```

this is a little bit cleaner, the `or` operator is triggered if the key doesn't exist.
```suggestion
      providerSourceAddress = data.provider-source-address or fallbackProviderSourceAddress;
```

---

Much nicer :heart:. I've incorporated the changes.